x/session/types/session_header.go

```diff
@@ -20,8 +17,12 @@ func (sh *SessionHeader) ValidateBasic() error {
 	}
 
 	// Validate the service
-	if sh.Service == nil || !sharedhelpers.IsValidService(sh.Service) {
-		return ErrSessionInvalidService.Wrapf("invalid service: %s", sh.Service)
+	if sh.Service == nil {
+		return ErrSessionInvalidService.Wrapf("missing service: %s", sh.Service)
```

Comment:
If we know `sh.Service` is nil, I don't think we need to interpolate it here.

```suggestion
		return ErrSessionInvalidService.Wrapf("missing service")
```